accounts/abi/bind/bind.go

```diff
@@ -43,6 +43,28 @@ const (
 	LangObjC
 )
 
+func isKeyWord(arg string) bool {
+	switch arg {
+	case "range":
+	case "func":
+	case "make":
+	case "for":
+	case "new":
+	case "switch":
+	case "case":
+	case "var":
+	case "const":
+	case "type":
+	case "iota":
+	case "return":
+	case "struct":
+	default:
+		return false
+	}
+
```

Comment:
Please remove this blank line

Reply:
Golang has 25 keywords in total, should we list all of them?

```
break     default      func    interface  select
case      defer        go      map        struct
chan      else         goto    package    switch
const     fallthrough  if      range      type
continue  for          import  return     var
```

Reply:
I was too lazy, but you are right I should insert them all